### PR TITLE
update relationship between course and module forms

### PR DESCRIPTION
**Pull request by chazkiker2**
component state is updating properly between program, course, and module forms

### CODE REVIEW

**Comment by chazkiker2** [comment]
Forms are functional. A couple small bugs, and waiting on one change from backend. But once that change is in these forms will be entirely functional. MVP Release 2 HIT. 

Now it's time to refactor :)

File: src/components/pages/course-view/CourseViewContainer.jsx [hunk 1]
```diff
@@ -3,7 +3,11 @@ import { Card } from 'antd';
 import { ModuleView } from '../';
 import { useDispatch, useSelector } from 'react-redux';
 import { coursesActions } from '../../../state/ducks/coursesDuck';
+/*
```
Comment (kai-blt) [inline src/components/pages/course-view/CourseViewContainer.jsx:6]: Perhaps this is extra information that you didn't want to include in this file???

**Review by kai-blt — CHANGES_REQUESTED** [review]
Hey Chaz, it looks like some extra comments made it into CourseViewContainer component. Did you mean to include this info here? If not, could you please remove it from the file, if you did intend to include it, perhaps some extra info as to why it is there would be great!

File: src/components/pages/course-view/CourseViewContainer.jsx (hunk 1 above)
Comment (chazkiker2) [reply src/components/pages/course-view/CourseViewContainer.jsx:6]: lol so funny! yup, thanks for the catch

**Pull request merged**